From 70318d6bcf7ef89e1a73fc775d33f4c854aed484 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Tue, 14 Apr 2020 14:49:19 +0200
Subject: [PATCH] src/dpdk_telemetry.c: fix build on musl

Include unistd.h instead of sys/unistd.h to fix the following build
failure on musl:

src/dpdk_telemetry.c:43:10: fatal error: sys/unistd.h: No such file or directory
 #include <sys/unistd.h>
          ^~~~~~~~~~~~~~

Fixes:
 - http://autobuild.buildroot.org/results/aafb8c72f147fefc7a988c45e4dc17de48b07a95

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: https://github.com/collectd/collectd/pull/3439]
---
 src/dpdk_telemetry.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dpdk_telemetry.c b/src/dpdk_telemetry.c
index b46e8574..cfee5f12 100755
--- a/src/dpdk_telemetry.c
+++ b/src/dpdk_telemetry.c
@@ -40,7 +40,7 @@
 #include <sys/queue.h>
 #include <sys/socket.h>
 #include <sys/un.h>
-#include <sys/unistd.h>
+#include <unistd.h>
 
 #define BUF_SIZE 100000
 #define PLUGIN_NAME "dpdk_telemetry"
-- 
2.25.1

